MAINTAINERS.md

```diff
@@ -0,0 +1,21 @@
+The current maintainers of this repo are [@adamralph](https://github.com/adamralph), [@bording](https://github.com/bording), and [@WojcikMike](https://github.com/WojcikMike).
+
+The maintainers [watch](https://github.com/Particular/NServiceBus.RabbitMQ/watchers) this repo and undertake the following responsibilities:
+
+- Ensuring that the `develop` branch is always releasable. "Releasable" means that the software built from the latest commit can be released immediately simply by following the release procedure.
```

Comment:
I don't see a problem with it. Committing something to develop that breaks the build or otherwise leaves it in a bad state seems like something to avoid.